apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/AbstractResourceTypeV2.java

```diff
@@ -90,6 +103,27 @@ public ResourceTransformer<TResource> getTransformer() {
     return transformer;
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * <p>If either phase or phase sequence number are null in the claim they are represented as zero
+   * in the version string.
+   *
+   * @param entityObject object of type TEntity
+   * @return a valid semantic version number
+   */
+  @Override
+  public String createPhaseVersionString(Object entityObject) {
```

Comment:
No it had to be an Object because of java type reasons around the fact we're processing captured wildcard types in the resource provider.  Based on your other suggestion though this might become cleaner even with it being Object.